Documentation/ApiOverview/Typo3CoreEngine/UsingDataHandler/Index.rst

```diff
@@ -129,7 +129,7 @@ Imagine the $data array something like this:
    );
 
 This aims to create two new pages in the page with uid "456". In the
-follow code this is submitted to the database. Notice how line 3
+following code this is submitted to the database. Notice how line 2
```

Comment:
Here I'd directly mention `$dataHandler->reverseOrder = 1` instead, then we don't need to fix this over and over again. It also allows for understanding this without jumping to the code example.

Reply:
I removed the line numbers altogether in this section, the code is self-explanatory.